ccl/backupccl: TestDataDriven failed

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4728159&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4728159&tab=artifacts#/) on master @ [dce0a307cd242d8575f59a153026b1af9e6a7936](https://github.com/cockroachdb/cockroach/commits/dce0a307cd242d8575f59a153026b1af9e6a7936):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.10s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
/cc @cockroachdb/bulk-io
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-14502

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4741219&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4741219&tab=artifacts#/) on master @ [e4fcae1625a35917d08a2a569d24529b0cc089d2](https://github.com/cockroachdb/cockroach/commits/e4fcae1625a35917d08a2a569d24529b0cc089d2):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.43s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4755815&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4755815&tab=artifacts#/) on master @ [fa0291175ecb983301d980247ad2ec6c7b233660](https://github.com/cockroachdb/cockroach/commits/fa0291175ecb983301d980247ad2ec6c7b233660):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:220: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3475/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.28s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4766705&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4766705&tab=artifacts#/) on master @ [0e6878b6140cb5cc7c799fdf576b3d565a54fd2e](https://github.com/cockroachdb/cockroach/commits/0e6878b6140cb5cc7c799fdf576b3d565a54fd2e):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.88s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4771642&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4771642&tab=artifacts#/) on master @ [71e32a6a5369dabde22043ca6ebbd11651404e5d](https://github.com/cockroachdb/cockroach/commits/71e32a6a5369dabde22043ca6ebbd11651404e5d):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.01s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4777940&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4777940&tab=artifacts#/) on master @ [9a66e9ad26bc34250bee89854cbce1999ea70f9f](https://github.com/cockroachdb/cockroach/commits/9a66e9ad26bc34250bee89854cbce1999ea70f9f):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (12.83s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4793239&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4793239&tab=artifacts#/) on master @ [63ea9139e2ca996e38b5fe7c7b43a97e625242f5](https://github.com/cockroachdb/cockroach/commits/63ea9139e2ca996e38b5fe7c7b43a97e625242f5):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.16s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4807921&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4807921&tab=artifacts#/) on master @ [72b71b6920fe5b6dde2c7229b4edc2f95432839e](https://github.com/cockroachdb/cockroach/commits/72b71b6920fe5b6dde2c7229b4edc2f95432839e):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4046/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (11.89s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4820483&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4820483&tab=artifacts#/) on master @ [9bff1de7115984957978416d829feb582eb2664e](https://github.com/cockroachdb/cockroach/commits/9bff1de7115984957978416d829feb582eb2664e):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3473/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.81s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4832036&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4832036&tab=artifacts#/) on master @ [7a97bdd02c62787f40d79d5a2cd3439d8c9c8a1b](https://github.com/cockroachdb/cockroach/commits/7a97bdd02c62787f40d79d5a2cd3439d8c9c8a1b):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (12.83s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4844150&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4844150&tab=artifacts#/) on master @ [ea4c75e28696ada91135346eade10366b1642ae4](https://github.com/cockroachdb/cockroach/commits/ea4c75e28696ada91135346eade10366b1642ae4):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.16s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4851463&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4851463&tab=artifacts#/) on master @ [bb739c7f9ca341096edf865d2abb85e76e7528ca](https://github.com/cockroachdb/cockroach/commits/bb739c7f9ca341096edf865d2abb85e76e7528ca):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (11.85s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4867776&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4867776&tab=artifacts#/) on master @ [9b80166bc5f3d3d1212c44a9e1ac219d458af9e0](https://github.com/cockroachdb/cockroach/commits/9b80166bc5f3d3d1212c44a9e1ac219d458af9e0):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.62s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4878223&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4878223&tab=artifacts#/) on master @ [43457fc6e2f5324823cca4156d9799453753b9cc](https://github.com/cockroachdb/cockroach/commits/43457fc6e2f5324823cca4156d9799453753b9cc):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.72s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4888523&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4888523&tab=artifacts#/) on master @ [cc07b5e7e670097560cb8412b380484773df1e96](https://github.com/cockroachdb/cockroach/commits/cc07b5e7e670097560cb8412b380484773df1e96):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.61s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4897945&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4897945&tab=artifacts#/) on master @ [b99291249d249ede65e732f89d3b3df8bf5472a4](https://github.com/cockroachdb/cockroach/commits/b99291249d249ede65e732f89d3b3df8bf5472a4):


```
=== RUN   TestDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestDataDriven3497569274
    test_log_scope.go:80: use -show-logs to present logs inline
    datadriven_test.go:296: Mock HTTP Storage "http://127.0.0.1:40435"
=== CONT  TestDataDriven
    datadriven_test.go:606: -- test log scope end --
--- FAIL: TestDataDriven (68.78s)
=== RUN   TestDataDriven/mismatched-localities
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3164/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:1:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3164/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:4:
        exec-sql [0 args]
        CREATE DATABASE d PRIMARY REGION "us-east-1" REGIONS "us-west-1";
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3164/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:8:
        exec-sql [0 args]
        BACKUP INTO (
          'nodelocal://1/default?COCKROACH_LOCALITY=default',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-west-1');
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3164/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:14: BACKUP INTO LATEST IN (
          'nodelocal://1/default?COCKROACH_LOCALITY=default',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-west-1',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-east-1');
        expected:
        pq: Requested backup has localities [region=us-west-1 region=us-east-1], but a previous backup layer in this collection has localities [region=us-west-1]. Mismatched backup layers are not supported. Please take a new full backup with the new localities, or an incremental backup with matching localities.
        
        found:
        pq: Requested backup has localities [region=us-east-1 region=us-west-1], but a previous backup layer in this collection has localities [region=us-west-1]. Mismatched backup layers are not supported. Please take a new full backup with the new localities, or an incremental backup with matching localities.
    --- FAIL: TestDataDriven/mismatched-localities (1.11s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4897946&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4897946&tab=artifacts#/) on master @ [b99291249d249ede65e732f89d3b3df8bf5472a4](https://github.com/cockroachdb/cockroach/commits/b99291249d249ede65e732f89d3b3df8bf5472a4):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3477/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (12.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4907709&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4907709&tab=artifacts#/) on master @ [771432d1099e516dbc11827c5458886c176e73e3](https://github.com/cockroachdb/cockroach/commits/771432d1099e516dbc11827c5458886c176e73e3):


```
=== RUN   TestDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestDataDriven1505858943
    test_log_scope.go:80: use -show-logs to present logs inline
    datadriven_test.go:296: Mock HTTP Storage "http://127.0.0.1:46879"
=== CONT  TestDataDriven
    datadriven_test.go:606: -- test log scope end --
--- FAIL: TestDataDriven (66.82s)
=== RUN   TestDataDriven/mismatched-localities
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:1:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:4:
        exec-sql [0 args]
        CREATE DATABASE d PRIMARY REGION "us-east-1" REGIONS "us-west-1";
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:8:
        exec-sql [0 args]
        BACKUP INTO (
          'nodelocal://1/default?COCKROACH_LOCALITY=default',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-west-1');
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:14: BACKUP INTO LATEST IN (
          'nodelocal://1/default?COCKROACH_LOCALITY=default',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-west-1',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-east-1');
        expected:
        pq: Requested backup has localities [region=us-west-1 region=us-east-1], but a previous backup layer in this collection has localities [region=us-west-1]. Mismatched backup layers are not supported. Please take a new full backup with the new localities, or an incremental backup with matching localities.
        
        found:
        pq: Requested backup has localities [region=us-east-1 region=us-west-1], but a previous backup layer in this collection has localities [region=us-west-1]. Mismatched backup layers are not supported. Please take a new full backup with the new localities, or an incremental backup with matching localities.
    --- FAIL: TestDataDriven/mismatched-localities (1.03s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4907710&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4907710&tab=artifacts#/) on master @ [771432d1099e516dbc11827c5458886c176e73e3](https://github.com/cockroachdb/cockroach/commits/771432d1099e516dbc11827c5458886c176e73e3):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.35s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4913542&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4913542&tab=artifacts#/) on master @ [dcb06e12a77c0a7f7fe9d2609c54327345987dcd](https://github.com/cockroachdb/cockroach/commits/dcb06e12a77c0a7f7fe9d2609c54327345987dcd):


```
=== RUN   TestDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestDataDriven1162566041
    test_log_scope.go:80: use -show-logs to present logs inline
    datadriven_test.go:296: Mock HTTP Storage "http://127.0.0.1:36529"
=== CONT  TestDataDriven
    datadriven_test.go:606: -- test log scope end --
--- FAIL: TestDataDriven (72.27s)
=== RUN   TestDataDriven/mismatched-localities
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:1:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:4:
        exec-sql [0 args]
        CREATE DATABASE d PRIMARY REGION "us-east-1" REGIONS "us-west-1";
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:8:
        exec-sql [0 args]
        BACKUP INTO (
          'nodelocal://1/default?COCKROACH_LOCALITY=default',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-west-1');
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:14: BACKUP INTO LATEST IN (
          'nodelocal://1/default?COCKROACH_LOCALITY=default',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-west-1',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-east-1');
        expected:
        pq: Requested backup has localities [region=us-west-1 region=us-east-1], but a previous backup layer in this collection has localities [region=us-west-1]. Mismatched backup layers are not supported. Please take a new full backup with the new localities, or an incremental backup with matching localities.
        
        found:
        pq: Requested backup has localities [region=us-east-1 region=us-west-1], but a previous backup layer in this collection has localities [region=us-west-1]. Mismatched backup layers are not supported. Please take a new full backup with the new localities, or an incremental backup with matching localities.
    --- FAIL: TestDataDriven/mismatched-localities (0.79s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4913543&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4913543&tab=artifacts#/) on master @ [dcb06e12a77c0a7f7fe9d2609c54327345987dcd](https://github.com/cockroachdb/cockroach/commits/dcb06e12a77c0a7f7fe9d2609c54327345987dcd):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.02s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4916430&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4916430&tab=artifacts#/) on master @ [6c91da48b8da78c13e4cb8675a1a98c5ec75ea06](https://github.com/cockroachdb/cockroach/commits/6c91da48b8da78c13e4cb8675a1a98c5ec75ea06):


```
=== RUN   TestDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestDataDriven3390960540
    test_log_scope.go:80: use -show-logs to present logs inline
    datadriven_test.go:296: Mock HTTP Storage "http://127.0.0.1:41311"
=== CONT  TestDataDriven
    datadriven_test.go:606: -- test log scope end --
--- FAIL: TestDataDriven (73.57s)
=== RUN   TestDataDriven/mismatched-localities
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:1:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:4:
        exec-sql [0 args]
        CREATE DATABASE d PRIMARY REGION "us-east-1" REGIONS "us-west-1";
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:8:
        exec-sql [0 args]
        BACKUP INTO (
          'nodelocal://1/default?COCKROACH_LOCALITY=default',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-west-1');
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3738/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/mismatched-localities:14: BACKUP INTO LATEST IN (
          'nodelocal://1/default?COCKROACH_LOCALITY=default',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-west-1',
          'nodelocal://1/database_backup?COCKROACH_LOCALITY=region%3Dus-east-1');
        expected:
        pq: Requested backup has localities [region=us-west-1 region=us-east-1], but a previous backup layer in this collection has localities [region=us-west-1]. Mismatched backup layers are not supported. Please take a new full backup with the new localities, or an incremental backup with matching localities.
        
        found:
        pq: Requested backup has localities [region=us-east-1 region=us-west-1], but a previous backup layer in this collection has localities [region=us-west-1]. Mismatched backup layers are not supported. Please take a new full backup with the new localities, or an incremental backup with matching localities.
    --- FAIL: TestDataDriven/mismatched-localities (0.91s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4916431&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4916431&tab=artifacts#/) on master @ [6c91da48b8da78c13e4cb8675a1a98c5ec75ea06](https://github.com/cockroachdb/cockroach/commits/6c91da48b8da78c13e4cb8675a1a98c5ec75ea06):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4051/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4924219&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4924219&tab=artifacts#/) on master @ [1fdbb16fa206c5fd77c3aa111ec40916edfb55df](https://github.com/cockroachdb/cockroach/commits/1fdbb16fa206c5fd77c3aa111ec40916edfb55df):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.04s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4935203&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4935203&tab=artifacts#/) on master @ [7ea67fd5d4966508877fc88ba21bdd498ff162e8](https://github.com/cockroachdb/cockroach/commits/7ea67fd5d4966508877fc88ba21bdd498ff162e8):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4053/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4949881&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4949881&tab=artifacts#/) on master @ [3553a79001c0e362f03eaae514c1d403764bd4c3](https://github.com/cockroachdb/cockroach/commits/3553a79001c0e362f03eaae514c1d403764bd4c3):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.81s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4962374&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4962374&tab=artifacts#/) on master @ [f6afa10a35de90a3c9e0dcc69b7680d24b39a4cb](https://github.com/cockroachdb/cockroach/commits/f6afa10a35de90a3c9e0dcc69b7680d24b39a4cb):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4050/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.70s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4976226&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4976226&tab=artifacts#/) on master @ [0add8527e82d0ae4987ea7105778c7aa6e43bd3a](https://github.com/cockroachdb/cockroach/commits/0add8527e82d0ae4987ea7105778c7aa6e43bd3a):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4054/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4054/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4054/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4054/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4054/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4054/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4984369&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4984369&tab=artifacts#/) on master @ [911ebee04c0230f1145375340f81745931727162](https://github.com/cockroachdb/cockroach/commits/911ebee04c0230f1145375340f81745931727162):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.69s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4992445&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4992445&tab=artifacts#/) on master @ [8fe42c18e5e2a82494fa6a34e068f4ab801f87c4](https://github.com/cockroachdb/cockroach/commits/8fe42c18e5e2a82494fa6a34e068f4ab801f87c4):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.32s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5004014&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5004014&tab=artifacts#/) on master @ [36b190afcffec66272bd5485567757b1a106c9e0](https://github.com/cockroachdb/cockroach/commits/36b190afcffec66272bd5485567757b1a106c9e0):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4058/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.66s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5019243&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5019243&tab=artifacts#/) on master @ [2fdc0a62df08829dd96901fffb5fb1fbb0b0997d](https://github.com/cockroachdb/cockroach/commits/2fdc0a62df08829dd96901fffb5fb1fbb0b0997d):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4070/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4070/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4070/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4070/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4070/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4070/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.97s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5032995&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5032995&tab=artifacts#/) on master @ [72e485705aa859ef850ebc67568bc29fe9b7cfd6](https://github.com/cockroachdb/cockroach/commits/72e485705aa859ef850ebc67568bc29fe9b7cfd6):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4073/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4073/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4073/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4073/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4073/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4073/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5048535&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5048535&tab=artifacts#/) on master @ [97e72bacbeb9574f09f7475a62ef45c3e228183e](https://github.com/cockroachdb/cockroach/commits/97e72bacbeb9574f09f7475a62ef45c3e228183e):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4075/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4075/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4075/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4075/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4075/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4075/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.89s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5060922&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5060922&tab=artifacts#/) on master @ [a2e1910f51593bd2ef72e1d7c615e08f95791186](https://github.com/cockroachdb/cockroach/commits/a2e1910f51593bd2ef72e1d7c615e08f95791186):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.64s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5068477&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5068477&tab=artifacts#/) on master @ [c9e0194b19a03d55c6be92572aad3fbafc256334](https://github.com/cockroachdb/cockroach/commits/c9e0194b19a03d55c6be92572aad3fbafc256334):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (15.51s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5073020&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5073020&tab=artifacts#/) on master @ [7a9eb906ce86e2f75db637e29d46cd6604fca7b4](https://github.com/cockroachdb/cockroach/commits/7a9eb906ce86e2f75db637e29d46cd6604fca7b4):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4086/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5082930&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5082930&tab=artifacts#/) on master @ [01572daaf94f80f81f843723a8b58d80fe128990](https://github.com/cockroachdb/cockroach/commits/01572daaf94f80f81f843723a8b58d80fe128990):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4090/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4090/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4090/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4090/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4090/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4090/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.70s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5095747&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5095747&tab=artifacts#/) on master @ [b01e1f3b3db311603dc7dd63647a8a737602143e](https://github.com/cockroachdb/cockroach/commits/b01e1f3b3db311603dc7dd63647a8a737602143e):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.65s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5109399&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5109399&tab=artifacts#/) on master @ [587302906426907122deae6eae5f68630d57e900](https://github.com/cockroachdb/cockroach/commits/587302906426907122deae6eae5f68630d57e900):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (15.68s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5120752&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5120752&tab=artifacts#/) on master @ [98bdf3241028c9b1bdff429fb455e61870adc9d0](https://github.com/cockroachdb/cockroach/commits/98bdf3241028c9b1bdff429fb455e61870adc9d0):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.67s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5129244&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5129244&tab=artifacts#/) on master @ [a86e6c9c4b82cb404e4a20fa70092823fd4a9439](https://github.com/cockroachdb/cockroach/commits/a86e6c9c4b82cb404e4a20fa70092823fd4a9439):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (15.57s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5134572&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5134572&tab=artifacts#/) on master @ [60d0f95ba057dab8d4f50d2903c504fe275d061a](https://github.com/cockroachdb/cockroach/commits/60d0f95ba057dab8d4f50d2903c504fe275d061a):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.35s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5139690&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5139690&tab=artifacts#/) on master @ [1a91d7cb7b93dfef5dcaf872125875cefa3e0190](https://github.com/cockroachdb/cockroach/commits/1a91d7cb7b93dfef5dcaf872125875cefa3e0190):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5143195&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5143195&tab=artifacts#/) on master @ [7f3c06f5f2c26bc84705430a3622f92ec1444e9d](https://github.com/cockroachdb/cockroach/commits/7f3c06f5f2c26bc84705430a3622f92ec1444e9d):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.15s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5149070&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5149070&tab=artifacts#/) on master @ [bc1ee7c7c276984fce8ff5ba4fcfcdff335dde50](https://github.com/cockroachdb/cockroach/commits/bc1ee7c7c276984fce8ff5ba4fcfcdff335dde50):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (15.47s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5152845&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5152845&tab=artifacts#/) on master @ [9a179ea1aa8d6723fb12a988f2212ac8493e5dfc](https://github.com/cockroachdb/cockroach/commits/9a179ea1aa8d6723fb12a988f2212ac8493e5dfc):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (15.68s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5157920&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5157920&tab=artifacts#/) on master @ [16c484aa84d3718bfc82557f8e935ab78e6753b6](https://github.com/cockroachdb/cockroach/commits/16c484aa84d3718bfc82557f8e935ab78e6753b6):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.65s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5163997&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5163997&tab=artifacts#/) on master @ [bc5f5b7159ed2b7dbffa94bb2a590fdddb642b9b](https://github.com/cockroachdb/cockroach/commits/bc5f5b7159ed2b7dbffa94bb2a590fdddb642b9b):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (12.78s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5176354&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5176354&tab=artifacts#/) on master @ [2c264bd12c15b8811553adc9d90f9230065f3ff3](https://github.com/cockroachdb/cockroach/commits/2c264bd12c15b8811553adc9d90f9230065f3ff3):


```
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:60:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
        pq: detected a mismatch in regions between the restore cluster and the backup cluster, missing regions detected: us-east-1, us-west-1.
        HINT: there are two ways you can resolve this issue: 1) update the cluster to which you're restoring to ensure that the regions present on the nodes' --locality flags match those present in the backup image, or 2) restore with the "skip_localities_check" option
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:67:
        exec-sql [0 args]
        CREATE DATABASE no_region_db;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:71:
        exec-sql [0 args]
        CREATE TABLE no_region_db.t (x INT);
        INSERT INTO no_region_db.t VALUES (1), (2), (3);
        CREATE DATABASE no_region_db_2;
        CREATE TABLE no_region_db_2.t (x INT);
        INSERT INTO no_region_db_2.t VALUES (1), (2), (3);
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:79:
        exec-sql [0 args]
        BACKUP DATABASE no_region_db INTO 'nodelocal://1/no_region_database_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:83:
        exec-sql [0 args]
        BACKUP INTO 'nodelocal://1/no_region_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:87:
        exec-sql [0 args]
        DROP DATABASE no_region_db;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:91:
        exec-sql [0 args]
        DROP DATABASE no_region_db_2;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:95: SET CLUSTER SETTING sql.defaults.primary_region = 'non-existent-region';
        expected:
        NOTICE: setting global default sql.defaults.primary_region is not recommended
        HINT: use the ``ALTER ROLE ... SET`` syntax to control session variable defaults at a finer-grained level. See: https://www.cockroachlabs.com/docs/v22.1/alter-role.html#set-default-session-variable-values-for-a-role
        
        found:
        NOTICE: setting global default sql.defaults.primary_region is not recommended
        HINT: use the ``ALTER ROLE ... SET`` syntax to control session variable defaults at a finer-grained level. See: https://www.cockroachlabs.com/docs/dev/alter-role.html#set-default-session-variable-values-for-a-role
    --- FAIL: TestDataDriven/multiregion (13.26s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5176356&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5176356&tab=artifacts#/) on master @ [2c264bd12c15b8811553adc9d90f9230065f3ff3](https://github.com/cockroachdb/cockroach/commits/2c264bd12c15b8811553adc9d90f9230065f3ff3):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (13.69s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5190227&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5190227&tab=artifacts#/) on master @ [a35b848edb3f5562199b0f56bc27a6c576073d5d](https://github.com/cockroachdb/cockroach/commits/a35b848edb3f5562199b0f56bc27a6c576073d5d):


```
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:60:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
        pq: detected a mismatch in regions between the restore cluster and the backup cluster, missing regions detected: us-east-1, us-west-1.
        HINT: there are two ways you can resolve this issue: 1) update the cluster to which you're restoring to ensure that the regions present on the nodes' --locality flags match those present in the backup image, or 2) restore with the "skip_localities_check" option
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:67:
        exec-sql [0 args]
        CREATE DATABASE no_region_db;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:71:
        exec-sql [0 args]
        CREATE TABLE no_region_db.t (x INT);
        INSERT INTO no_region_db.t VALUES (1), (2), (3);
        CREATE DATABASE no_region_db_2;
        CREATE TABLE no_region_db_2.t (x INT);
        INSERT INTO no_region_db_2.t VALUES (1), (2), (3);
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:79:
        exec-sql [0 args]
        BACKUP DATABASE no_region_db INTO 'nodelocal://1/no_region_database_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:83:
        exec-sql [0 args]
        BACKUP INTO 'nodelocal://1/no_region_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:87:
        exec-sql [0 args]
        DROP DATABASE no_region_db;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:91:
        exec-sql [0 args]
        DROP DATABASE no_region_db_2;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3778/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/multiregion:95: SET CLUSTER SETTING sql.defaults.primary_region = 'non-existent-region';
        expected:
        NOTICE: setting global default sql.defaults.primary_region is not recommended
        HINT: use the ``ALTER ROLE ... SET`` syntax to control session variable defaults at a finer-grained level. See: https://www.cockroachlabs.com/docs/v22.1/alter-role.html#set-default-session-variable-values-for-a-role
        
        found:
        NOTICE: setting global default sql.defaults.primary_region is not recommended
        HINT: use the ``ALTER ROLE ... SET`` syntax to control session variable defaults at a finer-grained level. See: https://www.cockroachlabs.com/docs/dev/alter-role.html#set-default-session-variable-values-for-a-role
    --- FAIL: TestDataDriven/multiregion (16.89s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5190229&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5190229&tab=artifacts#/) on master @ [a35b848edb3f5562199b0f56bc27a6c576073d5d](https://github.com/cockroachdb/cockroach/commits/a35b848edb3f5562199b0f56bc27a6c576073d5d):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3518/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3518/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3518/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3518/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3518/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:304: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3518/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (14.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5202471&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5202471&tab=artifacts#/) on master @ [ce66acdbba801de88f1dd645eaedeb3834f23dbd](https://github.com/cockroachdb/cockroach/commits/ce66acdbba801de88f1dd645eaedeb3834f23dbd):


```
        exec-sql [0 args]
        USE d1
        ----
    datadriven_test.go:307: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:98:
        query-sql [0 args]
        SELECT
          regexp_replace(schema_name, 'pg_temp.*', 'pg_temp') AS name
        FROM [SHOW SCHEMAS] ORDER BY name
        ----
        crdb_internal
        information_schema
        pg_catalog
        pg_extension
        public
    datadriven_test.go:307: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:109:
        query-sql [0 args]
        USE d1;
        SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
        ----
        perm_table
    datadriven_test.go:307: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:115:
        new-server [3 args]
        <no input to command>
        ----
    datadriven_test.go:307: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:118:
        exec-sql [0 args]
        USE defaultdb;
        ----
    datadriven_test.go:307: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:122:
        exec-sql [0 args]
        RESTORE FROM LATEST IN 'nodelocal://0/full_cluster_backup/';
        ----
    datadriven_test.go:307: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/backupccl/testdata/backup-restore/temp-tables:127: USE d1;
        SELECT schema_name FROM [SHOW SCHEMAS] ORDER BY schema_name
        output didn't match expected:
        @@ -1,7 +1,6 @@
         crdb_internal
         information_schema
         pg_catalog
         pg_extension
        -pg_temp_0_0
         public
         
    --- FAIL: TestDataDriven/temp-tables (16.56s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/backupccl/datadriven_test.go
+++ b/pkg/ccl/backupccl/datadriven_test.go
@@ -31,6 +31,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/datadriven"
 	"github.com/cockroachdb/errors"
 	"github.com/lib/pq"
@@ -70,22 +71,26 @@ type sqlDBKey struct {
 }
 
 type datadrivenTestState struct {
-	servers           map[string]serverutils.TestServerInterface
-	dataDirs          map[string]string
-	sqlDBs            map[sqlDBKey]*gosql.DB
-	jobTags           map[string]jobspb.JobID
-	clusterTimestamps map[string]string
-	noticeBuffer      []string
-	cleanupFns        []func()
+	servers map[string]serverutils.TestServerInterface
+	// tempObjectCleanupAndWait is a mapping from server name to a method that can
+	// be used to nudge and wait for temporary object cleanup.
+	tempObjectCleanupAndWait map[string]func()
+	dataDirs                 map[string]string
+	sqlDBs                   map[sqlDBKey]*gosql.DB
+	jobTags                  map[string]jobspb.JobID
+	clusterTimestamps        map[string]string
+	noticeBuffer             []string
+	cleanupFns               []func()
 }
 
 func newDatadrivenTestState() datadrivenTestState {
 	return datadrivenTestState{
-		servers:           make(map[string]serverutils.TestServerInterface),
-		dataDirs:          make(map[string]string),
-		sqlDBs:            make(map[sqlDBKey]*gosql.DB),
-		jobTags:           make(map[string]jobspb.JobID),
-		clusterTimestamps: make(map[string]string),
+		servers:                  make(map[string]serverutils.TestServerInterface),
+		tempObjectCleanupAndWait: make(map[string]func()),
+		dataDirs:                 make(map[string]string),
+		sqlDBs:                   make(map[sqlDBKey]*gosql.DB),
+		jobTags:                  make(map[string]jobspb.JobID),
+		clusterTimestamps:        make(map[string]string),
 	}
 }
 
@@ -103,13 +108,18 @@ func (d *datadrivenTestState) cleanup(ctx context.Context) {
 }
 
 type serverCfg struct {
-	name                 string
-	iodir                string
-	tempCleanupFrequency string
-	nodes                int
-	splits               int
-	ioConf               base.ExternalIODirConfig
-	localities           string
+	name  string
+	iodir string
+	// nudgeTempObjectsCleanup is a channel used to nudge the temporary object
+	// reconciliation job to run.
+	nudgeTempObjectsCleanup chan time.Time
+	// tempObjectCleanupDone is the channel used by the temporary object
+	// reconciliation job to signal it is done cleaning up.
+	tempObjectCleanupDone chan struct{}
+	nodes                 int
+	splits                int
+	ioConf                base.ExternalIODirConfig
+	localities            string
 }
 
 func (d *datadrivenTestState) addServer(t *testing.T, cfg serverCfg) error {
@@ -121,17 +131,21 @@ func (d *datadrivenTestState) addServer(t *testing.T, cfg serverCfg) error {
 		JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals(),
 	}
 
+	// If the server needs to control temporary object cleanup, let us set that up
+	// now.
+	if cfg.nudgeTempObjectsCleanup != nil && cfg.tempObjectCleanupDone != nil {
+		params.ServerArgs.Knobs.SQLExecutor = &sql.ExecutorTestingKnobs{
+			OnTempObjectsCleanupDone: func() {
+				cfg.tempObjectCleanupDone <- struct{}{}
+			},
+			TempObjectsCleanupCh: cfg.nudgeTempObjectsCleanup,
+		}
+	}
+
 	settings := cluster.MakeTestingClusterSettings()
 	closedts.TargetDuration.Override(context.Background(), &settings.SV, 10*time.Millisecond)
 	closedts.SideTransportCloseInterval.Override(context.Background(), &settings.SV, 10*time.Millisecond)
-	if cfg.tempCleanupFrequency != "" {
-		duration, err := time.ParseDuration(cfg.tempCleanupFrequency)
-		if err != nil {
-			return errors.New("unable to parse tempCleanupFrequency during server creation")
-		}
-		sql.TempObjectCleanupInterval.Override(context.Background(), &settings.SV, duration)
-		sql.TempObjectWaitInterval.Override(context.Background(), &settings.SV, time.Millisecond)
-	}
+	sql.TempObjectWaitInterval.Override(context.Background(), &settings.SV, time.Millisecond)
 	params.ServerArgs.Settings = settings
 
 	clusterSize := cfg.nodes
@@ -154,9 +168,25 @@ func (d *datadrivenTestState) addServer(t *testing.T, cfg serverCfg) error {
 		tc, _, cleanup = backupRestoreTestSetupEmptyWithParams(t, clusterSize, cfg.iodir,
 			InitManualReplication, params)
 	}
+	cleanupFn := func() {
+		if cfg.nudgeTempObjectsCleanup != nil {
+			close(cfg.nudgeTempObjectsCleanup)
+		}
+		if cfg.tempObjectCleanupDone != nil {
+			close(cfg.tempObjectCleanupDone)
+		}
+		cleanup()
+	}
 	d.servers[cfg.name] = tc.Server(0)
 	d.dataDirs[cfg.name] = cfg.iodir
-	d.cleanupFns = append(d.cleanupFns, cleanup)
+	d.cleanupFns = append(d.cleanupFns, cleanupFn)
+
+	if cfg.nudgeTempObjectsCleanup != nil && cfg.tempObjectCleanupDone != nil {
+		d.tempObjectCleanupAndWait[cfg.name] = func() {
+			cfg.nudgeTempObjectsCleanup <- timeutil.Now()
+			<-cfg.tempObjectCleanupDone
+		}
+	}
 
 	return nil
 }
@@ -214,9 +244,6 @@ func (d *datadrivenTestState) getSQLDB(t *testing.T, server string, user string)
 //
 //   + disable-http: disables use of external HTTP endpoints.
 //
-//   + temp-cleanup-freq: specifies the frequency with which the temporary table
-//   cleanup reconciliation job runs
-//
 //   + localities: specifies the localities that will be used when starting up
 //   the test cluster. The cluster will have len(localities) nodes, with each
 //   node assigned a locality config corresponding to the locality. Please
@@ -225,6 +252,10 @@ func (d *datadrivenTestState) getSQLDB(t *testing.T, server string, user string)
 //   + nodes: specifies the number of nodes in the test cluster.
 //
 //   + splits: specifies the number of ranges the bank table is split into.
+//
+//   + control-temp-object-cleanup: sets up the server in a way that the test
+//   can control when to run the temporary object reconciliation loop using
+//   nudge-and-wait-for-temp-cleanup
 //
 // - "exec-sql [server=<name>] [user=<name>] [args]"
 //   Executes the input SQL query on the target server. By default, server is
@@ -289,6 +320,9 @@ func (d *datadrivenTestState) getSQLDB(t *testing.T, server string, user string)
 //
 //   + expect-pausepoint: expects the schema change job to end up in a paused state because
 //   of a pausepoint error.
+//
+// - "nudge-and-wait-for-temp-cleanup"
+//    Nudges the temporary object reconciliation loop to run, and waits for completion.
 func TestDataDriven(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
@@ -307,24 +341,16 @@ func TestDataDriven(t *testing.T) {
 		datadriven.RunTest(t, path, func(t *testing.T, d *datadriven.TestData) string {
 
 			switch d.Cmd {
-			case "sleep":
-				var sleepDuration string
-				d.ScanArgs(t, "time", &sleepDuration)
-				duration, err := time.ParseDuration(sleepDuration)
-				if err != nil {
-					return err.Error()
-				}
-				time.Sleep(duration)
-				return ""
-
 			case "reset":
 				ds.cleanup(ctx)
 				ds = newDatadrivenTestState()
 				return ""
 
 			case "new-server":
-				var name, shareDirWith, iodir, tempCleanupFrequency, localities string
+				var name, shareDirWith, iodir, localities string
 				var splits int
+				var nudgeTempObjectCleanup chan time.Time
+				var tempObjectCleanupDone chan struct{}
 				nodes := singleNode
 				var io base.ExternalIODirConfig
 				d.ScanArgs(t, "name", &name)
@@ -340,9 +366,6 @@ func TestDataDriven(t *testing.T) {
 				if d.HasArg("disable-http") {
 					io.DisableHTTP = true
 				}
-				if d.HasArg("temp-cleanup-freq") {
-					d.ScanArgs(t, "temp-cleanup-freq", &tempCleanupFrequency)
-				}
 				if d.HasArg("localities") {
 					d.ScanArgs(t, "localities", &localities)
 				}
@@ -352,15 +375,21 @@ func TestDataDriven(t *testing.T) {
 				if d.HasArg("splits") {
 					d.ScanArgs(t, "splits", &splits)
 				}
+				if d.HasArg("control-temp-object-cleanup") {
+					nudgeTempObjectCleanup = make(chan time.Time)
+					tempObjectCleanupDone = make(chan struct{})
+				}
+
 				lastCreatedServer = name
 				cfg := serverCfg{
-					name:                 name,
-					iodir:                iodir,
-					tempCleanupFrequency: tempCleanupFrequency,
-					nodes:                nodes,
-					splits:               splits,
-					ioConf:               io,
-					localities:           localities,
+					name:                    name,
+					iodir:                   iodir,
+					nudgeTempObjectsCleanup: nudgeTempObjectCleanup,
+					tempObjectCleanupDone:   tempObjectCleanupDone,
+					nodes:                   nodes,
+					splits:                  splits,
+					ioConf:                  io,
+					localities:              localities,
 				}
 				err := ds.addServer(t, cfg)
 				if err != nil {
@@ -603,6 +632,15 @@ func TestDataDriven(t *testing.T) {
 				err := ds.getSQLDB(t, server, user).QueryRow(`SELECT cluster_logical_timestamp()`).Scan(&ts)
 				require.NoError(t, err)
 				ds.clusterTimestamps[timestampTag] = ts
+				return ""
+
+			case "nudge-and-wait-for-temp-cleanup":
+				server := lastCreatedServer
+				if nudgeAndWait, ok := ds.tempObjectCleanupAndWait[server]; !ok {
+					t.Fatalf("server %s was not configured with `control-temp-object-cleanup`", server)
+				} else {
+					nudgeAndWait()
+				}
 				return ""
 
 			default:

--- a/pkg/ccl/backupccl/testdata/backup-restore/temp-tables
+++ b/pkg/ccl/backupccl/testdata/backup-restore/temp-tables
@@ -112,7 +112,7 @@ SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
 ----
 perm_table
 
-new-server name=s2 share-io-dir=s1 temp-cleanup-freq=5s
+new-server name=s2 share-io-dir=s1 control-temp-object-cleanup
 ----
 
 exec-sql
@@ -155,8 +155,8 @@ perm_table
 temp_seq
 temp_table
 
-# Wait for the temp cleanup job to run.
-sleep time=5s
+# Nudge and wait for the temp cleanup job to run.
+nudge-and-wait-for-temp-cleanup
 ----
 
 # The synthetic temp schema should have been erased.

